app/webpacker/controllers/paymentmethod_controller.js

```diff
@@ -12,6 +12,14 @@ export default class extends Controller {
 
   selectPaymentMethod(event) {
     this.doSelectPaymentMethod(event.target.dataset.paymentmethodId);
+    const stripeCardsController =
+      this.application.getControllerForElementAndIdentifier(
+        document
+          .getElementById(event.target.dataset.paymentmethodId)
+          .querySelector('[data-controller="stripe-cards"]'),
+        "stripe-cards"
+      );
+    stripeCardsController.connect();
```

Comment:
What problem is this solving? :thinking:

Reply:
When selecting a payment method (such as Stripe), we want to re-initialize the stripe-cards controller in order select the card and then add the `disabled` attribute to each `input` of the `stripeelements` container.
Thus, we won't submit those input if we select an already registered card.

_(thanks for having a look here, it needs another review on my side, but I wanted to be sure to have your opinion here since you've made a lot here)_

Reply:
Ah I see, it's because we disable all the fields in the preceding line...